.github/workflows/build-docker-image.yaml

```
@@ -0,0 +1,152 @@
+    inputs:
+      source_branch:
+        required: true
+        type: string
+      version_type:
+        required: true
+        type: string
+        description: "Type of version to build: commit, pr, release, rc"
+      architecture:
+        required: true
+        type: string
+        default: "x86_64-linux"
```

actionlint (1.7.9)

input "architecture" of workflow_call event has the default value "x86_64-linux", but it is also required. if an input is marked as required, its default value will never be used

(events)